lib/elixir/lib/kernel.ex

```diff
@@ -2799,8 +2867,11 @@ defmodule Kernel do
   end
 
   defp build_unless(_condition, _arguments) do
-    raise(ArgumentError, "invalid or duplicate keys for unless, only \"do\" " <>
-      "and an optional \"else\" are permitted")
+    raise(
+      ArgumentError,
+      "invalid or duplicate keys for unless, only \"do\" " <>
```

Comment:
Please assign the message to a variable in the previous line and also remove the parens around raise.